problem1/concrete/item.go

```diff
@@ -0,0 +1,104 @@
+package concrete
```

Comment:
add .gitignore
add output files(if needed) to ouput folder
change structuring of packages

Reply:
add linter https://golangci-lint.run/usage/install/

Reply:
added .gitignore
added different folder for input and output sample files
changed structuring of packages (modified for better names)